source/Octo.Tests/Integration/IntegrationTestBase.cs

```diff
@@ -131,177 +131,5 @@ public void Configure(IApplicationBuilder app)
                 app.UseOwin(x => x.UseNancy());
             }
         }
-
-
-
-        #region Nancy JSON Serializers
-        public class JsonNetBodyDeserializer : IBodyDeserializer
```

Comment:
These weren't used.

Reply:
Turns out they were used... by magic reflection stuff in nancy. Removing them broke the tests